labextension/src/index.ts

```diff
@@ -105,6 +175,15 @@ async function activate(
     },
   });
 
+  commands.addCommand(CommandIDs.open, {
+    execute: (args) => {
+      const model = args["sp"] as IServerProxyModel;
+      const url = PageConfig.getBaseUrl() + model.url;
```

Comment:
Generally, one should use the `URLExt.join` feature instead of string concatenation/templating when working with URLs.

Reply:
Observed to be resolved now